### PR TITLE
SLM-231: Correct sign out link based on whether using magic link or one time code

### PR DESCRIPTION
**Pull request by nathanrussell-moj-digital**
PR to use the correct sign-out link based on whether we are using magic link auth or one time code auth

### CODE REVIEW

File: server/app.ts [hunk 1]
```diff
@@ -83,6 +84,9 @@ export default function createApp(
   app.get('/privacy-policy', (req, res) => res.render('pages/privacy-policy/privacy-policy'))
   app.use('/start', setupLegalSenderStartPage())
   app.use('/contact-helpdesk', setupContactHelpdesk(zendeskService))
+  app.use('/legal-sender/sign-out', (req, res) =>
```
Comment (nathanrussell-moj-digital) [inline server/app.ts:87]: Create a new sign-out route that simply redirects to the correct route (with `&force=true` appended)

File: server/views/partials/header.njk [hunk 2]
```diff
@@ -36,7 +36,7 @@
           </li>
 
           <li class="hmpps-header__navigation__item">
-            <a data-qa="signOut" class="hmpps-header__link" href="/link/request-link?force=true">Sign out</a>
+            <a data-qa="signOut" class="hmpps-header__link" href="/legal-sender/sign-out">Sign out</a>
```
Comment (nathanrussell-moj-digital) [inline server/views/partials/header.njk:39]: Use the new route to sign out, which in turn redirects to the correct sign out route

**Review by joelstobart-moj — APPROVED** [review]
LGTM

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request merged**